### ISSUE TITLE
Further check rangeFromLineNumber parameters

### ISSUE THREAD

**Issue by Arcanemagus** [AtomLinter/atom-linter#59]
Currently being triggered by the following steps in `linter-eslint`, but stopping it here will fix anywhere else that this is causing crashes on:

1. Try to run `linter-eslint` with a module missing ("parser `babel-eslint` requested but not installed
2. Get a response from the linter of:
```json
{
  "message": "Cannot find module 'babel-eslint'",
  "line": 0,
  "column": 0,
  "severity": 2
}
```
3\. `Helpers.rangeFromLineNumber(textEditor, line - 1)` is then called, meaning `rangeFromLineNumber` is being called with a line of `-1`.

So, tl;dr: Verify that `lineNumber` and `colStart` are within the valid range for a line before attempting to call Atom functions.
